src/allotropy/allotrope/models/plate_reader_benchling_2023_09_plate_reader.py

```diff
@@ -359,7 +359,7 @@ class CalculatedDataDocumentItem:
 
 @dataclass
 class CalculatedDataAggregateDocument:
-    calculated_data_document: list[CalculatedDataDocumentItem]
+    calculated_data_document: Optional[list[CalculatedDataDocumentItem]] = None
```

Comment:
`calculated data document`(s) should not be optional. If there is a `calculated data aggregate document` (which is optional), it should have at least one `calculated data document`.